### DISCUSSION TITLE
Decreased values when exported to excel

### DISCUSSION THREAD

**Discussion by AParisotto** [Open-Systems-Pharmacology/Forum#847] [Questions & Problems] [answered]
Hi everyone
When the concentration profile versus time is exported to Excel, the observed values are divided again. For example, a value in nanogram / milliliter is divided by some number (maybe 1 million, but I'm not sure) can I solve this problem just multiply the values again by some specific number? Or did any of you go through this and how did you prevent it?

**Comment by msevestre** [comment]
@AParisotto 
A few points to clarify:
 * Are your talking about PKSim or MoBi? 
* What are you exporting to excel? a simulation or a plot?
* Are you talking about CSV export maybe? 

Cheers,

**Comment by AParisotto** [comment]
sorry for the question wording
Clarify
I'm referring to PK-SIM
I am exporting the results of the concentration vs time graph
I mean CVS

**Comment by msevestre** [comment, marked as answer]
CSV are always exported in the internal unit so that other tools using the results (for example R toolbox) do not have to do any conversation

Concentration values are exported in umol/l
